deployment/config/public-graphql-api-gateway/gateway.config.ts

```diff
@@ -0,0 +1,15 @@
+// @ts-expect-error not a dependency
+import { defineConfig } from '@graphql-hive/gateway';
+
+export const gatewayConfig = defineConfig({
+  transportEntries: {
```

Comment:
Ah, @n1ru4l , now that this gw is running, you can hook it with otel and we'll have better insights on how our public API is being used (as separate traces from the internal one). 

So maybe it is worth considering OTEL plugin here, with OTEL context-propagation. 

Should be as simple as: 
```ts
  openTelemetry: {
    exporters: [
      createOtlpHttpExporter({
        url: '<get-this-one-from-Pulumi>'
      })
    ]
  }
```

Reply:
cc @EmrysMyrddin